### ISSUE TITLE
a11y: ensure that panels can be moved and resized with the keyboard

### ISSUE THREAD

**Issue by taleena** [grafana/scenes#1088]
Keyboard users are unable to interact with the move panel functionality, and that this is documented. Also please validate that this below earlier report no longer is a factor.

From a report in a non-scenes version: It's worth noting the interaction changes dependent on if the panel has a title or not (whether the icon appears besides the menu or it creates a header area to drag).

**Comment by taleena** [comment]
This is wcag-level/A
